x-pack/plugins/rollup/public/visualize/editor_config.js

```diff
@@ -54,16 +58,16 @@ export function initEditorConfig() {
     if (aggTypeName === 'date_histogram') {
       const interval = fieldAgg.interval;
       return {
-        interval: {
-          fixedValue: 'custom',
-        },
         useNormalizedEsInterval: {
           fixedValue: false,
         },
-        customInterval: {
+        interval: {
           default: interval,
           timeBase: interval,
-          help: `Must be a multiple of rollup configuration interval: ${interval}`
+          help: i18n.translate('xpack.rollupJobs.editorConfig.dateHistogram.customInterval.helpText', {
```

Comment:
@jen-huang I updated the config and added translations for help texts here. Could you please review?

Reply:
I tested the functionality specific to rollups (histogram interval and date histogram interval inputs) and it is working well 👍 

During testing I also found that some of the inputs have a strange space between the value and cursor. In this gif you can see that the space is added after I submit change to date histogram interval. The space doesn't appear to be text-selectable:

![May-01-2019 10-25-15](https://user-images.githubusercontent.com/1965714/57031107-99a1ec80-6bfb-11e9-9796-c532e96f4a04.gif)